quiz.js

```diff
@@ -0,0 +1,69 @@
+ // Loop over the inventory and populate the page
+
+ function populatePage(inventory) {
+    var carsContainer = document.getElementById('containerRow') //identify DOM element to hold car objects in inventory
+    var purchaseStatus
+    inventory.forEach(function(car) { //set purchase status based on boolean in car object
+        if (car.purchased === true) {
+            purchaseStatus = "Available!"
+        } else {
+            purchaseStatus = "Sold out!"
+        }
+
+        containerRow.innerHTML += // append each object to targeted DOM element
+        `<div class="col-md-3" style= "border: 5px solid ${car.color}; box-shadow: 10px 5px 7px black"  id="allCars">
+        <h3 class="carColumn">${car.make} ${car.model}</h3>
+        <p class="carColumn">${car.year}
+        <p class="carColumn">&#36;${car.price}
+        <p class="carColumn">${car.color}
+        <p class="carColumn" id="carDescription">${car.description}
+        <p class="carColumn" style = "font-weight: bold">${purchaseStatus}
+        </p></p></p></p></p></div>`
+    })
+}
+CarLot.loadInventory(populatePage);
+// CarLot.getInventory(console.log)
+
+  // Now that the DOM is loaded, establish all the event listeners needed
+
+  function activateEvents(inventory) {
```

Comment:
This `activateEvents()` should be in it's own IIFE and it should have all the event handlers with it.